lib/ui/search.dart

```diff
@@ -0,0 +1,223 @@
+      bottomNavigationBar: const BottomBar(),
+      appBar: AppBar(
+        title: Text('Mail Search',  style:
+        TextStyle(fontWeight: _commonFontWeight, fontSize: _commonFontSize),
+        ),
+        backgroundColor: _buttonColor,
+        centerTitle: true,
+      ),
+      body:
+      SingleChildScrollView(
+        child:
+        Container(
+          padding: EdgeInsets.all(15.0),
+          child: Column(
+              children: [
+                SfDateRangePicker(
+                  selectionMode: DateRangePickerSelectionMode.range,
+                  onSelectionChanged: (DateRangePickerSelectionChangedArgs args) {
+                    final DateTime rangeStartDate;
+                    final DateTime rangeEndDate;
+                    final dynamic value = args.value;
+                    if (args.value is PickerDateRange) {
+                      rangeStartDate = value.startDate;
+                      rangeEndDate = value.endDate == null ? value.startDate : value.endDate;
+                    }
+                    else {
+                      rangeStartDate = _today;
+                      rangeEndDate = _today;
+                    }
+                    setState(() {
+                      _start = rangeStartDate;
+                      _end = rangeEndDate;
+                    });
+                  },
+                  showNavigationArrow: true,
+                  maxDate: _today,
+                  rangeSelectionColor: Color.fromRGBO(51, 51, 102, 100.0),
+                  startRangeSelectionColor: Color.fromRGBO(51, 51, 102, 1.0),
+                  endRangeSelectionColor: Color.fromRGBO(51, 51, 102, 1.0),
+                  todayHighlightColor: Color.fromRGBO(231, 25, 33, 1.0),
+                ),
+                Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      Expanded(
+                        child:
+                        Padding(padding: EdgeInsets.only(right:5.0),
+                          child:
+                          Column(
+                              crossAxisAlignment: CrossAxisAlignment.stretch,
+                              children:[
+                                Text(
+                                  "Start Date:",
+                                  style: TextStyle(
+                                      fontSize: _buttonLabelTextSize,
+                                      fontWeight: _commonFontWeight,
+                                      color: Color.fromRGBO(51, 51, 102, 100)
+                                  ),
+                                ),
+                                SizedBox(
+                                  height: 50.0,
+                                  child:
+                                  ElevatedButton.icon(
+                                    style: ElevatedButton.styleFrom(
+                                      backgroundColor: Color.fromRGBO(51, 51, 102, 1.0),
```

Comment:
I'm getting an error with this backgroundColor saying there is no named parameter

Reply:
@shaneknows I was just looking into that and it works well on my side with no errors

Reply:
Weird I see it working with the build artifact, but I cannot get this to work locally without changing this to primary